src/adapter/src/coord/sequencer/inner/create_continual_task.rs

```diff
@@ -110,6 +110,7 @@ impl Coordinator {
         let read_holds = self.acquire_read_holds(&id_bundle);
         let as_of = read_holds.least_valid_read();
         physical_plan.set_as_of(as_of.clone());
+        physical_plan.set_initial_as_of(as_of.clone());
```

Comment:
This is not related to/required by the rest of the PR, right?

Reply:
Yup. Added a comment here